storage/innobase/xtrabackup/src/backup_mysql.cc

```diff
@@ -2053,7 +2045,6 @@ bool write_backup_config_file() {
     << innodb_checksum_algorithm_names[srv_checksum_algorithm] << "\n"
     << "innodb_log_checksums=" << srv_log_checksums << "\n"
     << "innodb_data_file_path=" << innobase_data_file_path << "\n"
-    << "innodb_log_files_in_group=" << srv_n_log_files << "\n"
```

Comment:
for pre-8.0.30, shouldn't the conifg file contain innodb_log_files_in_group.

Similarly, shouldn't we save the new innodb_redo_log_capacity in the config for 8.0.30 server restore?

Reply:
infact, we might want to retain the config for 8.0.30 too if the user is still using the deprecated parameter

Reply:
@rahulmalik87 @altmannmarcelo when do we use this .cnf file?

Reply:
@satya-bodapati we use them at --prepare. We needed this config before to create the correct number of files on the second time we start innodb during --prepare. We don't create the files anymore (we delegate it to the server that will restore the backup) so its safe to remove this.